redirects.js

```diff
@@ -109,11 +109,6 @@ function redirects() {
       destination: "/dashboard/settings/devices",
       permanent: false,
     },
-    {
```

Comment:
if we're removing this redirect please make sure *nothing* anywhere in the world still links to it

or alternatively instead of removing it redirect it to the proper new place :)


also while talking about redirects: if we're changing any previous templates urls then make sure to add redirects for them!